src/raiden_libs/blockchain.py

```diff
@@ -147,14 +148,17 @@ def get_blockchain_events(
 
     events: List[Event] = []
     for event_dict in registry_events:
+        token_network_address = TokenNetworkAddress(
+            decode_hex(event_dict["args"]["token_network_address"])
```

Comment:
Again, `to_canonical_address` would be more consistent then using `decode_hex` which is just possible due to the typecast.